firebase_admin/messaging.py

```diff
@@ -233,10 +353,14 @@ def __init__(self, app):
                 'Project ID is required to access Cloud Messaging service. Either set the '
                 'projectId option, or use service account credentials. Alternatively, set the '
                 'GOOGLE_CLOUD_PROJECT environment variable.')
-        self._fcm_url = _MessagingService.FCM_URL.format(project_id)
+        self._fcm_service = build('fcm', 'v1', credentials=app.credential.get_credential())
```

Comment:
I can't seem to get this to auth by using `credentials`. I've only been able to get it to work with a `developerKey`. I've yet to poke around the `google-api-python-client` library to see what exactly is happening. Any ideas?

Reply:
The error I'm getting is `googleapiclient.errors.HttpError: <HttpError 403 when requesting https://fcm.googleapis.com/$discovery/rest?version=v1 returned "The request is missing a valid API key.">`

Reply:
I played around with the googleapiclient a bit. I'll post my findings in a separate reply.